src/puppetlabs/puppetdb/constants.clj

```diff
@@ -0,0 +1,4 @@
+(ns puppetlabs.puppetdb.constants)
+
+(def filename-forbidden-characters
```

Comment:
We don't have to worry about UTF-8 encoding accidentally producing ASCII bytes. It's one of the design goals of UTF-8 that it doesn't: all multi-byte sequences that encode bytes > 127 have the high bit set in every byte, so they'll never contain ASCII bytes.